plugins/main/public/components/common/wazuh_discover/config/chart.ts

```diff
@@ -0,0 +1,130 @@
+import { DashboardPanelState } from "../../../../../../../src/plugins/dashboard/public/application";
+import { EmbeddableInput } from "../../../../../../../src/plugins/embeddable/public";
+
+const hitsHistogram = (indexPatternId: string) => {
+    return {
+        id: 'most_detected_vulnerabilities',
+        title: 'Most detected vulnerabilities',
+        type: 'horizontal_bar',
+        params: {
+            type: 'histogram',
```

Comment:
plugins/main/public/components/common/wazuh_discover/config/chart.ts This file should not be in common as it is very specific to Vulnerabilities.

Reply:
Thanks @chantal-kelm